services/wallet/database.go

```diff
@@ -0,0 +1,254 @@
+package wallet
+
+import (
+	"database/sql"
+	"database/sql/driver"
+	"encoding/json"
+	"errors"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/status-im/status-go/services/wallet/migrations"
+	"github.com/status-im/status-go/sqlite"
+)
+
+const (
+	errNoRows = "sql: no rows in result set"
+)
+
+// InitializeDB creates db file at a given path and applies migrations.
+func InitializeDB(path string) (*Database, error) {
+	db, err := sqlite.OpenDB(path)
+	if err != nil {
+		return nil, err
+	}
+	err = migrations.Migrate(db)
+	if err != nil {
+		return nil, err
+	}
+	return &Database{db: db}, nil
+}
+
+// SQLBigInt type for storing uint256 in the databse.
+// FIXME(dshulyak) SQL big int is max 64 bits. Maybe store as bytes in big endian and hope
+// that lexographical sorting will work.
+type SQLBigInt big.Int
+
+// Scan implements interface.
+func (i *SQLBigInt) Scan(value interface{}) error {
+	val, ok := value.(int64)
+	if !ok {
+		return errors.New("not an integer")
+	}
+	(*big.Int)(i).SetInt64(val)
+	return nil
+}
+
+// Value implements interface.
+func (i *SQLBigInt) Value() (driver.Value, error) {
+	if !(*big.Int)(i).IsInt64() {
+		return nil, errors.New("not at int64")
+	}
+	return (*big.Int)(i).Int64(), nil
+}
+
+// JSONBlob type for marshaling/unmarshaling inner type to json.
+type JSONBlob struct {
+	Interface interface{}
```

Comment:
```suggestion
	data interface{}
```

Because underlying value shouldn't be really exposed and `data` because it better describes the value.

Reply:
done